src/DataViews/LiveFunctionsDataView.cpp

```diff
@@ -315,7 +316,7 @@ DataView::ActionStatus LiveFunctionsDataView::GetActionStatus(
   }
 
   for (int index : selected_indices) {
-    uint64_t instrumented_function_id = GetInstrumentedFunctionId(index);
+    uint64_t instrumented_function_id = GetScopeId(index);
```

Comment:
Is this a scope_id or a function_id?

Reply:
`scope_id` it is. Anyway, I was about to rewrite the method in the next PR.

Reply:
As of now it also dereferences null.